src/Reflection/Adapter/ReflectionObject.php

```diff
@@ -175,7 +175,9 @@ public function hasProperty($name)
     }
 
     /**
-     * {@inheritDoc}
+     * @param string $name
```

Comment:
We shouldn't change the parent API in the `Adapter` ns

Reply:
See https://github.com/Roave/BetterReflection/issues/389

Reply:
same as discussion in https://github.com/Roave/BetterReflection/pull/394/files#r164996850

Reply:
Handled in #391

Reply:
Reverted